portal-backend/depmap/settings/download_settings.py

```diff
@@ -17,6 +17,7 @@ def get_download_list() -> List[DownloadRelease]:
     # now these releases may include things that the current user is not allowed to see, so filter the list
     owner_ids = set(get_visible_owner_id_configs().keys())
 
+    # TODO: not sure what an access controlled release would be? I assume I don't need to worry about this?
     releases: List[DownloadRelease] = [
         release for release in download_releases if release.owner_id in owner_ids
```

Comment:
@pgm this looks like some sort of release-level access control which I don't believe we're using - but I'm also not that familiar with how the releases are organized in the portal. Do you know of any reason that this filtering should be kept?